### PR TITLE
Log resolvedUrl

### PR DESCRIPTION
**Pull request by sudoryan**
Log resolvedUrl for test domain on production

### CODE REVIEW

File: server/pages/[domain]/index.page.tsx [hunk 1]
```diff
@@ -1667,7 +1667,20 @@ export async function getServerSideProps(props: DomainProfileServerSideProps) {
 
   const udMeHostname = new URL(config.UD_ME_BASE_URL).hostname;
   if (domain === 'testingdotcom.com' || domain === '0xtesting.nft') {
-    console.log('Domain profile headers output:', JSON.stringify({headers: req.headers}));
+    console.log(
+      'Profile page getServerSideProps output:',
+      JSON.stringify(
```
Comment (qrtp) [inline server/pages/[domain]/index.page.tsx:1672]: Do we want to log the entire props? Like...

```
JSON.stringify(
        {
          props,
          headers: {
            ...req.headers,
          },
        },
        null,
        2,
      ),
```

**Review by qrtp — APPROVED** [review]

File: server/pages/[domain]/index.page.tsx (hunk 1 above)
Comment (sudoryan) [reply server/pages/[domain]/index.page.tsx:1672]: the props object contains some circular references that can't be stringified. I defined a few more params to print

**Pull request merged**